inc/modules/export/epub/class-epub201.php

```diff
@@ -928,8 +935,18 @@ protected function createTitle( $book_contents, $metadata ) {
 		} else {
 			$html .= sprintf( '<h1 class="title">%s</h1>', get_bloginfo( 'name' ) );
 			$html .= sprintf( '<h2 class="subtitle">%s</h2>', ( isset( $metadata['pb_subtitle'] ) ) ? $metadata['pb_subtitle'] : '' );
-			$html .= sprintf( '<h3 class="author">%s</h3>', ( isset( $metadata['pb_author'] ) ) ? $metadata['pb_author'] : '' );
-			$html .= sprintf( '<h4 class="author">%s</h4>', ( isset( $metadata['pb_contributing_authors'] ) ) ? $metadata['pb_contributing_authors'] : '' );
+			if ( isset( $metadata['pb_authors'] ) ) {
+				$authors = oxford_comma_explode( $metadata['pb_authors'] );
```

Comment:
We should be returning the comma-separated list in one containing element, not multiple containing elements. Desired display is like:

> Sharon Kioko and Justin Marlowe

Not:

> Sharon Kioko
> Justin Marlowe

See: https://press.rebus.community/financialstrategy/